### ISSUE TITLE
Phythin

### ISSUE THREAD

**Issue by Antonio5521** [s-e-knudsen/Protonpass#1]
Can you tell if Pythin is the same application as Python? If not, where can I get the Phythin application?

i did follow the steps in readme, unfortunately no output.csv file.

Thank you in advance

**Comment by s-e-knudsen** [comment]
yes sorry. It is written in Python3. 

What do happens when you do it. Can you descibe your process?

**Comment by s-e-knudsen** [comment]
> Can you tell if Pythin is the same application as Python? If not, where can I get the Phythin application?
> 
> i did follow the steps in readme, unfortunately no output.csv file.
> 
> Thank you in advance

yes sorry. It is written in Python3.

What do happens when you do it. Can you descibe your process?

**Comment by s-e-knudsen** [comment]
> Can you tell if Pythin is the same application as Python? If not, where can I get the Phythin application?
> 
> i did follow the steps in readme, unfortunately no output.csv file.
> 
> Thank you in advance

I have updated the readme now also.

**Comment by s-e-knudsen** [comment]
Have it helped or can i help in resolving the issue?

**Comment by Antonio5521** [comment]
Dear, 

Yes, thank you very much !
I did solve why it wasn't working, you request to change the .fsk file to import.json.
In the Python script you refer to the filename "input.json".

I changed the file name into input.json and I did the CSV file.
Which I was able to successfully import in Proton Pass. 

Thank you, I will inform Proton as I did as them for more then a year to have this kind of solution you did write in Phyton.

Regards
